pkg/kubelet/server/stats/types.go

```diff
@@ -45,6 +45,8 @@ type NodeStats struct {
 	Memory *MemoryStats `json:"memory,omitempty"`
 	// Stats pertaining to network resources.
 	Network *NetworkStats `json:"network,omitempty"`
+	// Stats pertaining to total usage of filesystem resources.
+	Fs []FsStats `json:"fs,omitempty"`
```

Comment:
Now that FsStats is not device-specific, I think this should be a pointer not a slice?